### DISCUSSION TITLE
Error: start_date doesn't work

### DISCUSSION THREAD

**Discussion by purplelunamine** [Monika-After-Story/MonikaModDev#8329] [QA - code and content creation (technical)] [answered]
Today I was coding my own song for fun, absolutely no reason. I used the code that's on the wiki for the date: ![image](https://user-images.githubusercontent.com/77215030/144293136-585d118c-053c-4936-970a-f03356ccf15b.png)

But it didn't work, the program says "invalid syntax". I don't know much about coding, so I don't really know what this could mean...
![image](https://user-images.githubusercontent.com/77215030/144293179-5e53349b-6248-404b-a3e3-88103083f318.png)

Does anyone know how to fix this? Thanks.

**Comment by multimokia** [comment, marked as answer]
Missing comma on the end of the `aff_range` line
